plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/WatchManager.java

```diff
@@ -53,7 +53,8 @@
 			ResourceKind.BUILD, 
 			ResourceKind.IMAGE_STREAM, 
 			ResourceKind.ROUTE,
-			ResourceKind.TEMPLATE
+			ResourceKind.TEMPLATE,
```

Comment:
@jcantrill does WatchManager ommit IMAGE_STREAM_TAG on purpose? Is WatchManager not observing these on purpose?